docs/readthedocs/source/doc/Nano/Overview/pytorch_inference.md

```diff
@@ -26,13 +26,19 @@ Before you go ahead with these APIs, you have to make sure BigDL-Nano is correct
 .. note::
     You can install all required dependencies by
 
-    ::
+    .. code-block:: bash
 
         pip install --pre --upgrade bigdl-nano[pytorch,inference]
 
     This will install all dependencies required by BigDL-Nano PyTorch inference.
```

Comment:
It's recommanded since it will install all dependencies required by BigDL-Nano PyTorch inference with no version conflict issue.

Reply:
Thank you for the suggestion! I have updated the installation box as suggested.